state/dynamodb.yaml

```diff
@@ -309,6 +325,60 @@ Resources:
       - {'Fn::ImportValue': !Sub '${ParentAlertStack}-TopicARN'}
       OKActions:
       - {'Fn::ImportValue': !Sub '${ParentAlertStack}-TopicARN'}
+  BackupVault:
+    Type: 'AWS::Backup::BackupVault'
+    Properties:
+      # AccessPolicy: # FIXME later
```

Comment:
@michaelwittig Do we need a resource policy?

Reply:
no